lib/Command/Configure.php

```diff
@@ -109,8 +109,10 @@ private function configureSignal(InputInterface $input, OutputInterface $output)
 
 	private function configureSms(InputInterface $input, OutputInterface $output) {
 		$helper = $this->getHelper('question');
-
-		$providerQuestion = new Question('Please choose a SMS provider (websms, playsms, clockworksms, plivo, puzzelsms, ecallsms, voipms, huawei_e3531, spryng, sms77io, ovh, clickatellcentral, clicksend): ', 'websms');
+		$providerArray = ['websms', 'playsms', 'clockworksms', 'puzzelsms', 'ecallsms', 'voipms', 'huawei_e3531', 'spryng', 'sms77io', 'ovh', 'clickatellcentral', 'clicksend', 'plivo'];
+		sort($providerArray);
+		$strOfProviders = implode(',',$providerArray);
```

Comment:
```
			$strOfProviders = implode(',', $providerArray);
			$providerQuestion = new Question('Please choose a SMS provider (' . $strOfProviders . '): ', 'websms');
```

Always place a space after a comma (I'm so sorry to be this petty) and before and after concetting.

Reply:
(Here it's a hit-n-hope - I don't really know if it is complying code style rules! Just my personal feeling)

Reply:
No problem, I'll run `composer cs:fix` on the Configure.php file. I just ran it on my Plivo* files.

Reply:
@boppy I updated the file with your suggested coding style edits.